conda/fetch.py

```diff
@@ -91,31 +82,48 @@ def fetch_repodata(url, cache_dir=None, use_cache=False, session=None):
     if use_cache:
         return cache
 
+    if not ssl_verify:
+        try:
+            from requests.packages.urllib3.connectionpool import InsecureRequestWarning
+        except ImportError:
+            pass
+        else:
+            warnings.simplefilter('ignore', InsecureRequestWarning)
+
+    session = session or CondaSession()
+
     headers = {}
     if "_etag" in cache:
         headers["If-None-Match"] = cache["_etag"]
     if "_mod" in cache:
         headers["If-Modified-Since"] = cache["_mod"]
+    if 'anaconda.org' in url:
+        headers['Accept-Encoding'] = 'gzip;q=1.0, identity;q=0.1'
```

Comment:
I personally would do the opposite here. That is: I would do `gzip, deflate, compress, identity` (no quality parameters) for everything *but* `continuum.io`.